### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#154] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-23 185010](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/e2377a79-2575-4a99-bf34-1d28f288a77f)

**Comment by Elia-Belli** [comment, marked as answer]
- **(1)** Se $A\cap B=\emptyset\Rightarrow \mathbb{P}(A \cup B)= \mathbb{P}(A)+\mathbb{P}(B)\rightarrow \mathbb{P}(B)=\mathbb{P}(A\cup B)-\mathbb{P}(A)=0.5-0.3=0.2$
- **(2)** Se $\mathbb{P}(A \cap B)=\mathbb{P}(A)\mathbb{P}(B)\Rightarrow \mathbb{P}(B)=\mathbb{P}(A \cap B)/\mathbb{P}(A)$
$A \cap B$ non necessariamente vuoto: $\mathbb{P}(A \cup B)= \mathbb{P}(A)+\mathbb{P}(B)-\mathbb{P}(A \cap B)\rightarrow \mathbb{P}(A \cap B)= -\mathbb{P}(A \cup B)+\mathbb{P}(A)+\mathbb{P}(B)$
$\Rightarrow \mathbb{P}(B)=(-\mathbb{P}(A \cup B)+\mathbb{P}(A)+\mathbb{P}(B))/\mathbb{P}(A)\rightarrow p=\frac{-\mathbb{P}(A \cup B)}{\mathbb{P}(A)}+1+\frac{p}{\mathbb{P}(A)}$
$\rightarrow p(1-1/0.3)=\frac{-0.5}{0.3}+1\rightarrow p=\frac{-0.2}{0.3}/\frac{-0.7}{0.3}=0.2/0.7\simeq 0.286$
- **(3)** Se $A \subseteq B\Rightarrow \mathbb{P}(A \cup B)=\mathbb{P}(B)\rightarrow p=0.5$

**Comment by daromacarlo** [comment]
![Immagine 2023-10-26 125727](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/b8098e95-7d5c-4a83-9b20-33f2d834ca52)
![Immagine 2023-10-26 125756](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/e680a296-243b-4363-bdeb-86012e420b6d)
